api_app/analyzers_manager/file_analyzers/iocfinder.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from ioc_finder import find_iocs
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+
+logger = logging.getLogger(__name__)
+
+
+class IocFinder(FileAnalyzer):
+    parse_domain_from_url: bool = (True,)
+    parse_from_url_path: bool = (True,)
+    parse_domain_from_email_address: bool = (True,)
+    parse_address_from_cidr: bool = (True,)
+    parse_domain_name_from_xmpp_address: bool = (True,)
+    parse_urls_without_scheme: bool = (True,)
+    parse_imphashes: bool = (True,)
+    parse_authentihashes: bool = (True,)
```

Comment:
maybe some linter corrected these booleans into tuples?

Reply:
yes, I think so, it missed my eyes too. correcting